Gemfile

```diff
@@ -85,14 +85,14 @@ group :development, :test do
   gem 'spring-commands-rspec'
   gem 'rb-fsevent' #osx file system changes
   gem 'factory_girl_rails', require: false
-  gem 'faker'
+  gem 'database_cleaner'
+  gem 'faker', :git => 'git://github.com/stympy/faker.git', :branch => 'master'
```

Comment:
@somedayrainbows can you add a comment here about why we need to use the forked version of this gem? Also please switch to Ruby 2+ style keyword hash (`git: "git:/...", branch: 'master'`)